src/migration/artifact.go

```diff
@@ -42,7 +42,7 @@ func abstractArtData(ctx context.Context) error {
 			continue
 		}
 		for _, repo := range repos {
-			log.Debugf("abstracting artifact metadata under repository %s ....", repo.Name)
+			log.Infof("abstracting artifact metadata under repository %s ....", repo.Name)
```

Comment:
Is it possible to add another log message like "listing artifacts for migration..."?
The list may take a long time.

Reply:
The artifact list is read from database